stratum/hal/lib/barefoot/bfrt_table_manager.cc

```diff
@@ -176,7 +176,9 @@ ::util::Status BfrtTableManager::BuildTableKey(
                                 ::p4::config::v1::MatchField::EXACT)
               << "Found match field of type EXACT does not fit match field "
               << expected_match_field.ShortDebugString() << ".";
-          CHECK_RETURN_IF_FALSE(!IsDontCareMatch(mk.exact()));
```

Comment:
I believe there is no "don't care" for exact match

Reply:
correct, that function will always return false.

Reply:
maybe add a comment :)